owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewVideoErrorAdapter.java

```diff
@@ -24,6 +24,7 @@
 import android.content.Context;
 
 import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.PlaybackException;
```

Comment:
This import is not used, is it? ktLint didn't detect this because it is a java file 🙈

Reply:
Good catch! Fixed.